trackr-app/app/src/main/java/com/trackr/trackr_app/usecases/UserManager.kt

```diff
@@ -0,0 +1,34 @@
+package com.trackr.trackr_app.usecases
+
+import com.trackr.trackr_app.entities.User
+import java.util.UUID
+
+/**
+ * UserManager manages users
+ */
+class UserManager {
+    /**
+     * Create a new user
+     * @param id - unique user identifier
+     * @param username - whatever the user wants to call themselves
+     */
+    fun createUser(
+            id: UUID,
+            username: String,
+    ): User {
+        return User(id, username)
+    }
+
+    /**
+     * Return a new User object with the same uuid but a different username
+     * @param user - the user to edit
+     * @param new_username - the new username
+     */
+    fun editUsername(
+            user: User,
+            new_username: String,
+    ): User {
+        return User(user.id, new_username)
```

Comment:
should edit involve returning the user? I think right now the user is not being edited/mutated and we're just creating a new instance of a user.

Reply:
This method was going to return a user that would be passed through to the database in another method. Alternatively, I could make it so the database edit occurs from this method. Is that preferable?